specs/utility/file.table

```diff
@@ -24,6 +24,9 @@ extended_schema(WINDOWS, [
     Column("volume_serial", TEXT, "Volume serial number"),
     Column("file_id", TEXT, "file ID"),
 ])
+extended_schema(DARWIN, [
+    Column("bsd_flags", TEXT, "The BSD file flags")
```

Comment:
This description doesn't add any context.

`Flags set for this file separated by commas.`

It might also make sense to enumerate what flags this column could contain

Reply:
I couldn't find a more specific names, as they are always referred as "file flags". Sometimes they are also called chflags, but that is the name of the command line tool used to set them.

I've added the list of possible values in the description